Index: src/remuco/Main.java
===================================================================
--- src/remuco/Main.java	(revision 91)
+++ src/remuco/Main.java	(working copy)
@@ -23,6 +23,9 @@
 
 public class Main extends MIDlet implements CommandListener {
 
+	private static final Command CMD_ALERT_DISMISS = new Command("Ok",
+			Command.OK, 1);
+
 	private static final Command CMD_BACK = new Command("Ok", Command.BACK, 10);
 
 	private static final Command CMD_SHOW_LOG = new Command("Logs",
@@ -108,7 +111,7 @@
 			notifyDestroyed();
 		} else if (c == CMD_BACK) { // back from log form
 			screenMain.setActive(true);
-		} else if (c == Alert.DISMISS_COMMAND) { // back from error alert
+		} else if (c == CMD_ALERT_DISMISS) { // back from error alert
 			// show the log which exits the app if the user says 'exit'
 			logForm.removeCommand(CMD_BACK);
 			logForm.addCommand(IScreen.CMD_DISPOSE);
@@ -271,6 +274,7 @@
 		alert.setType(AlertType.ERROR);
 		alert.setString(msg);
 		alert.setTimeout(Alert.FOREVER);
+		alert.addCommand(CMD_ALERT_DISMISS);
 		alert.setCommandListener(this);
 		display.setCurrent(alert);
 	}
Index: src/remuco/ui/WaitingScreen.java
===================================================================
--- src/remuco/ui/WaitingScreen.java	(revision 91)
+++ src/remuco/ui/WaitingScreen.java	(working copy)
@@ -1,25 +1,18 @@
 package remuco.ui;
 
 import javax.microedition.lcdui.Form;
-import javax.microedition.lcdui.Gauge;
-import javax.microedition.lcdui.Item;
 
 public class WaitingScreen extends Form {
 
-	private Gauge g;
-
 	public WaitingScreen() {
 		super("Remuco");
 
-		g = new Gauge("", false, Gauge.INDEFINITE, Gauge.CONTINUOUS_RUNNING);
-		g.setLayout(Item.LAYOUT_CENTER | Item.LAYOUT_VCENTER);
-		g.setLabel(" ");
-
-		append(g);
+		append(" ");
 	}
 
 	public void setMessage(String m) {
-		g.setLabel(m);
+		delete(0);
+		append(m);
 	}
 
 }
Index: build.properties
===================================================================
--- build.properties	(revision 91)
+++ build.properties	(working copy)
@@ -20,6 +20,6 @@
 # UI 'simple' should work on all devices
 #remuco.ui=simple
 
-#remuco-profile=legacy
-remuco-profile=simple
+remuco-profile=legacy
+#remuco-profile=simple
 #remuco-profile=canvas
